src/journal-remote/journal-gatewayd.c

```diff
@@ -56,14 +56,14 @@ STATIC_DESTRUCTOR_REGISTER(arg_file, strv_freep);
 typedef struct RequestMeta {
         sd_journal *journal;
 
-        OutputMode mode;
-
         char *cursor;
         usec_t since, until;
         int64_t n_skip;
         uint64_t n_entries;
         bool n_entries_set, since_set, until_set;
 
+        OutputMode mode;
```

Comment:
Also here

Reply:
fix it https://github.com/systemd/systemd/pull/37424/commits/5fddc47d7e96a069c3fd0fea4452d09bd1367508